junit-jupiter-params/junit-jupiter-params.gradle.kts

```diff
@@ -21,6 +22,11 @@ dependencies {
 	osgiVerification(projects.junitPlatformLauncher)
 }
 
+extraJavaModuleInfo {
+	automaticModule(libs.univocity.parsers, "univocity.parsers")
```

Comment:
:speech_balloon: This is used to add an `Automatic-Module-Name` manifest to the JAR so Gradle will put it on the module path.